app/view-plants/page.tsx

```diff
@@ -0,0 +1,26 @@
+import PlantCard from '@/components/PlantCard/PlantCard';
+import { Plant } from '@/types/schema';
+
+export default function Home() {
+  const plant: Plant = {
```

Comment:
can we also try using the `get_plant_by_id` query you wrote here to display the PlantCard!